static/_redirects

```diff
@@ -544,3 +544,7 @@
 # 10/05/23
 /build/dapp/smart-contracts/toolchains/truffle /deprecated/truffle 301
 /build/dapp/smart-contracts/verification/verify-truffle /deprecated/verify-truffle 301
+
+# 10/16/2023
+
+/nodes/run/with-installer /nodes/run/with-installer/preparing-your-environment
```

Comment:
please double check that the spanish path is also redirected. I am not positive that this redirect will cover it so you might need to add:
`/es/nodes/run/with-installer /es/nodes/run/with-installer/preparing-your-environment`
on line 551

Reply:
Hmm, it's not working!

Reply:
fixed